src/ResXManager.View/Visuals/TranslationItem.cs

```diff
@@ -44,14 +44,25 @@ public TranslationItem(ResourceTableEntry entry, string source, CultureKey targe
             set => _translation = value;
         }
 
-        public bool Apply(string? prefix)
+        public bool UpdateValue(string? prefix)
         {
             if (!_entry.CanEdit(TargetCulture))
                 return false;
 
             return _entry.Values.SetValue(TargetCulture, prefix + Translation);
         }
 
+        public bool UpdateComment(string? prefix)
+        {
+            var neutralLanguageComment = _entry.Comment;
+
+            if (neutralLanguageComment == null)

```

Comment:
if there is no comment yet, shouldn't that just add the prefix as a comment, rather than terminating?

Reply:
I added the null check as a safety check, but upon further investigation, this null check isn't needed, as the comment will never be null. The comment property is incorrectly marked as a nullable type, as it defaults to `string.Empty` instead of returning null.